src/Tasks/Microsoft.NET.Build.Extensions.Tasks/msbuildExtensions/Microsoft/Microsoft.NET.Build.Extensions/Microsoft.NET.Build.Extensions.NETFramework.targets

```diff
@@ -56,9 +64,19 @@ Copyright (c) .NET Foundation. All rights reserved.
     </PropertyGroup>
     <NETBuildExtensionsError Condition="'$(DependsOnNETStandard)' == 'true' AND '$(NETStandardInbox)' != 'true' AND '$(_UsingOldSDK)' == 'true'"
                              ResourceName="UnsupportedSDKVersionForNetStandard20"/>
+
+    <!--
+      .NET 4.7.1 has support for .NET Standard 2.0 built-in, so most of the facades aren't necessary.  However, the assembly versions of a few set of assemblies
+      do not yet match the ones shipped by the support package.  This means that the versions from the contract NuGet packages would be preferred to the in-box 
+      version (which is newer).  So if there is a dependency on netstandard.dll, we use the 4.2.0.0 version of the DLLs from the .NET Standard 2.0 "facades".
+      (Though these DLLs are not actually facades, they contain the implementation.)
+      -->
+    <ItemGroup Condition="'$(_TargetFrameworkVersionWithoutV)' == '4.7.1'">
+      <_NETStandardLibraryNETFrameworkLib Include="$(MSBuildThisFileDirectory)\net471\lib\*.dll" Condition="'$(DependsOnNETStandard)' == 'true'"/>
+    </ItemGroup>
     
     <!-- if any reference depends on netstandard and it is not inbox, add references and implementation assemblies for netstandard2.0  -->
-    <ItemGroup Condition="'$(DependsOnNETStandard)' == 'true' AND '$(NETStandardInbox)' != 'true'">
+    <ItemGroup Condition="'$(DependsOnNETStandard)' == 'true' AND '$(NETStandardInbox)' != 'true' AND '$(_TargetFrameworkVersionWithoutV)' &lt; '4.7.1'">
```

Comment:
> Or is this there in case people explicitly set NETStandardInbox to false in their 4.7.1 projects as a workaround for this issue?

Yes that is the case. Initially I had it just with `!=` but got a comment saying that in case people override that value, we should make sure this logic doesn't kick in for 4.7.2+